### ISSUE TITLE
Institution Content - COE

### ISSUE THREAD

**Issue by hellolynn-tbtb** [bcgov/SIMS#1779]
### **Describe the task**
- Lynn to implement content updates in DEV environment.
- Content updates include business feedback, accessibility (AA+), and B.C. government content standards.

### **Acceptance Criteria**
- [ ] Update COE table
- [ ] Update COE view
- [ ] Update modal view

**Comment by michesmith** [comment]
@astridSABC will create a new ticket once Jeremy is done his content review
